README.md

```diff
@@ -25,9 +25,9 @@ To build an older version of Cantaloupe, you can specify a commit hash:
 
     mvn verify -Dcantaloupe.version=dev -Dcantaloupe.commit.ref=fff5425
 
-To apply your own patches to the Cantaloupe source, create a patchfile in `src/main/docker/patches` with a Git diff, then use the following:
+To apply your own patches to the Cantaloupe source, create patchfiles with a Git diff, put them in a directory of your choosing, and then use the following (where `src/main/docker/patches` is the directory that contains the patchfiles to be applied):
```

Comment:
I know the implementation works for our purposes (since we plan to always use the `src/main/docker/patches` dir), but if the dir isn't actually configurable, then these usage instructions are not accurate.

Could we simplify the usage and say: to patch the application, make your desired modifications to `src/main/docker/patches` (add/remove files) and then pass `-Dcantaloupe.apply.patches=true`? That way the Dockerfile can always use the same patches directory.

Reply:
Yeah this is a cleaner and makes it less confusing. Will push up the change. Thanks!